privacy_evaluator/attacks/property_inference_attack/property_inference_data_augmentation_attack.py

```diff
@@ -0,0 +1,280 @@
+from torch.nn.modules import adaptive
+from ...classifiers.classifier import Classifier
+from ...utils import data_utils
+from ...output.user_output_property_inference_attack import (
+    UserOutputPropertyInferenceAttack,
+)
+from .property_inference_attack import PropertyInferenceAttack
+
+import numpy as np
+import logging
+from tqdm.auto import tqdm
+from typing import Tuple, Dict, List
+from collections import OrderedDict
+
+# count of shadow training sets, must be even
+AMOUNT_SETS = 2
+# ratio and size for unbalanced data sets
+SIZE_SHADOW_TRAINING_SET = 1000
+# ratios for different properties in sub-attacks
+RATIOS_FOR_ATTACK = [
+    0.05,
+    0.1,
+    0.15,
+    0.2,
+    0.25,
+    0.3,
+    0.35,
+    0.4,
+    0.45,
+    0.55,
+    0.6,
+    0.65,
+    0.7,
+    0.75,
+    0.8,
+    0.85,
+    0.9,
+    0.95,
+]
+
+# number of epochs for training the meta classifier
+NUM_EPOCHS_META_CLASSIFIER = 20
+
+# The type of adaptation. ('mask', 'random_noise', 'brightness')
+ADAPTATION = "mask"
```

Comment:
You are totally correct! For each adaptation I added a sentence.

Reply:
I can see your changes when I go under 'Files changed' and on the team branch. Hence, I don't think we have to give it too much attention since everything is correct otherwise.